.github/workflows/common_ci.yml

```diff
@@ -63,6 +63,7 @@ jobs:
           enable_persistence_tests: 'true'
           test_service_port: ${{ env.TEST_SERVICE_PORT }}
           token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
TODO: @keelerm84 , I've updated this to your contract test branch just to see it passing/failing in CI. When ready to merge this would ideally point to `v3`.

Reply:
Nevermind, I don't think you can point test harness at arbitrary branches using the action.